library/Hooks-windows.cpp

```diff
@@ -40,8 +41,8 @@ distribution.
 
 // we don't know which of the SDL functions will be called first... so we
 // just catch the first one and init all our function pointers at that time
-bool FirstCall(void);
-bool inited = false;
+void InitSDLPointers(void);
+std::once_flag inited;
```

Comment:
How about static linkage for variable and function?

Reply:
done